data/dataloader.py

```diff
@@ -233,14 +331,14 @@ def _get_image_dir(self, data_root):
         master_db_dir = '/data6/phytoplankton-db/hab_in_situ/hab_field'
         img_dir = os.path.basename(data_root).split('.')[0]
         img_dir = os.path.join(master_db_dir, img_dir)
+        img_dir = '/data6/phytoplankton-db/hab_in_situ/hab_field/field_20190701-0911/'
```

Comment:
raising issue to get rid of hard codings for data/dataloader.SPCHABDataset.check_SPCformat

Reply:
Functions like get image_dir will probably not be used since we are going to use sql in the future any way.